lib/stat_tracker.rb

```diff
@@ -239,37 +322,6 @@ def avg_team_goals_league
       team_avgs
   end
 
-  # Finally the actual methods
```

Comment:
Thank you for reorganizing these methods; it's a lot more intuitive and matches the iteration 2 more closely.